src/test/javascript/spec/app/admin/health/health.component.spec.ts

```diff
@@ -1,9 +1,10 @@
-import { ComponentFixture, TestBed, async } from '@angular/core/testing';
-import { MockBackend } from '@angular/http/testing';
-import { Http, BaseRequestOptions } from '@angular/http';
-import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
-import { JhiHealthCheckComponent } from '../../../../../../main/webapp/app/admin/health/health.component';
-import { JhiHealthService } from '../../../../../../main/webapp/app/admin/health/health.service';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
```

Comment:
Ahhh, I don't know why my Intellij formatter remove spaces sometimes... Now it's ok in all TypeScript files :)